### PR TITLE
Add Firmware Version to the log

### PR DESCRIPTION
**Pull request by Megamouse**
Adds the firmware version to the first line in the log:
RPCS3 v0.0.6-7804-046cc0c7 Alpha | get_firmware_version | Firmware version: 4.84

### CODE REVIEW

**Comment by uaqlover** [comment]
any updates ? nice to see merge to master

**Comment by Asinin3** [comment]
@uaqlover Don't spam PR's with the same "when will it merge" question, of course the devs want their PR's to be merged when they are ready. So unless it has been a considerable amount of time or you want to ask something specific it's basically just spam. Same deal with all the "Thanks" comments, use the GitHub Thumbs Up emoji/reaction. All the comments are sending out email notifications so GitHub isn't really the place for discussions. That's what our Discord is for https://discord.me/RPCS3

File: Utilities/Log.cpp [hunk 1]
```diff
@@ -598,7 +598,7 @@ logs::file_listener::file_listener(const std::string& name)
 	ver.m.ch  = nullptr;
 	ver.m.sev = level::always;
 	ver.stamp = 0;
-	ver.text = fmt::format("RPCS3 v%s | %s\n%s", rpcs3::version.to_string(), rpcs3::get_branch(), utils::get_system_info());
+	ver.text = fmt::format("RPCS3 v%s | %s | %s\n%s", rpcs3::version.to_string(), rpcs3::get_branch(), utils::get_firmware_version(), utils::get_system_info());
```
Comment (VelocityRa) [inline Utilities/Log.cpp]:
```suggestion
	ver.text = fmt::format("RPCS3 v%s | %s | Firmware Version: %s\n%s", rpcs3::version.to_string(), rpcs3::get_branch(), utils::get_firmware_version(), utils::get_system_info());
```

File: Utilities/sysinfo.cpp [hunk 2]
```diff
@@ -128,3 +130,26 @@ std::string utils::get_system_info()
...
+		const size_t start = version.find_first_of(':') + 1;
+		const size_t end = version.find_first_of(':', start);
+		version = version.substr(start, end - start);
+
+		// Trim version
+		const size_t trim_start = version.find_first_not_of('0');
+		const size_t trim_end = version.find_last_not_of('0');
+		version = version.substr(trim_start, trim_end);
+
+		return "Firmware version: " + version;
```
Comment (VelocityRa) [inline Utilities/sysinfo.cpp]:
```suggestion
		return version;
```

File: Utilities/Log.cpp (hunk 1 above)
Comment (Megamouse) [reply Utilities/Log.cpp]: better?

**Pull request merged**